sticker/src/tensorflow/tagger.rs

```diff
@@ -283,10 +304,46 @@ where
         Ok(labels)
     }
 
+    fn tag_sentences_top_k_(
+        &self,
+        sentences: &[impl Borrow<Sentence>],
+    ) -> Result<Vec<Vec<Vec<&T>>>, Error> {
+        let builder = self.prepare_batch(sentences)?;
+
+        // Tag the batch
+        let tag_tensor = self.tag_sequences(
+            builder.seq_lens(),
+            builder.inputs(),
+            &self.graph.top_k_predicted_op,
+        )?;
+
+        // Convert label numbers to labels.
+        let max_seq_len = tag_tensor.dims()[1] as usize;
+        let k = tag_tensor.dims()[2] as usize;
+        let mut labels = Vec::new();
+        for (idx, sentence) in sentences.iter().enumerate() {
+            let seq_len = min(max_seq_len, sentence.borrow().len());
+            let offset = idx * max_seq_len * k;
+            let seq = &tag_tensor[offset..offset + seq_len * k];
+
+            labels.push(
+                seq.iter()
+                    .chunks(k)
+                    .into_iter()
+                    .map(|c| c.map(|&label| self.labels.value(label as usize).unwrap()))
+                    .map(FromIterator::from_iter)
```

Comment:
Could be more verbose (and less flexible) as: `.map(Vec::from_iter)`, don't think it's necessary to change but might be easier for code readers.

Reply:
Sounds good.